README.md

```diff
@@ -62,16 +62,15 @@ Generate yaml based matrix and run job :muscle:
 
 ## Detail
 ### Excluding logic
-Excluding pattern may be specified with `List` of `Map` (e.g. `List<Map<String, String>>`)
-
+Excluding pattern may be specified with `List` of `Map` (e.g. `List<Map<String, [String or List]>>`)
+Elements in the `Map` may be a List to exclude multiple items for one key
```

Comment:
**[IMO]** I prefer to append  new paragraph here

```suggestion

Elements in the `Map` may be a List to exclude multiple items for one key
```

# Before
![image](https://user-images.githubusercontent.com/608755/99606210-a50e7000-2a4c-11eb-8510-f3094bb8268e.png)

# After
![image](https://user-images.githubusercontent.com/608755/99606225-ad66ab00-2a4c-11eb-893c-8bc8058e5c28.png)